cmd/rigs/cmd/publish_status.go

```diff
@@ -0,0 +1,25 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	publishCmd.AddCommand(statusCmd)
+}
+
+var statusCmd = &cobra.Command{
```

Comment:
Just a little command to check the storage status of a cid in web3/nft.storage manually. Made it to debug my work and figured I'd leave it.